config/utils_test.go

```diff
@@ -142,6 +142,36 @@ func TestFixInvalidLocales(t *testing.T) {
 	assert.Contains(t, *cfg.LocalizationSettings.AvailableLocales, *cfg.LocalizationSettings.DefaultClientLocale, "DefaultClientLocale should have been added to AvailableLocales")
 }
 
+func TestStripPassword(t *testing.T) {
```

Comment:
Looks great. Nice use of `for range` to avoid duplicating code.  
I would only add the (_not so_) edge case you mentioned where the password contains `:` and `@` chars.

Reply:
Good suggestion. Added them.